src/index.js

```diff
@@ -272,7 +272,9 @@ class LumigoPlugin {
 
 			for (const func of functions) {
 				const funcRuntime = func.runtime || runtime;
-				func.layers = func.layers || [];
```

Comment:
When a functionllevel layer property is defined, Serverless will ignore the provider-level property, so we need to add the layers here ourselves.